scripts/aem.js

```diff
@@ -675,6 +631,39 @@ async function waitForLCP(lcpBlocks) {
   });
 }
 
+/**
+ * Loads all blocks in a section.
+ * @param {Element} section The section element
+ */
+
+async function loadSection(section, loadCallback) {
+  const status = section.dataset.sectionStatus;
+  if (status === 'initialized') {
+    section.dataset.sectionStatus = 'loading';
+    const blocks = [...section.querySelectorAll('div.block')];
+    for (let i = 0; i < blocks.length; i += 1) {
+      // eslint-disable-next-line no-await-in-loop
+      await loadBlock(blocks[i]);
+    }
+    if (loadCallback) await loadCallback(section);
+    section.dataset.sectionStatus = 'loaded';
+    section.style.display = null;
+  }
+}
+
+/**
+ * Loads all sections.
+ * @param {Element} element The parent element of sections to load
+ */
+
+async function loadSections(element) {
+  const sections = [...element.querySelectorAll('div.section')];
+  for (let i = 0; i < sections.length; i += 1) {
```

Comment:
```
for (const [idx, section] of sections.entries()) {
  await loadSection(section);
}
```

Reply:
Linting does not like this approach...
![image](https://github.com/adobe/aem-boilerplate/assets/474200/bbe68025-dd47-4c49-8793-ca379afce2cf)